PersonApi.Tests/Readme.md

```diff
@@ -1,14 +1,29 @@
-# Running a local coverage report
```

Comment:
You updated the wrong readme. 
You should be updating the readme at the root of the repo, not this one on the tests project.
(Content was good though)

Reply:
:D figures I'd do something like that. I'll move it around. Thanks :)